Extension/MultipathUpgrade/Enum/CharacterClassType.cs

```diff
@@ -0,0 +1,11 @@
+﻿namespace YAPO.MultipathUpgrade.Enum
```

Comment:
It might be best to use a different namespace structure. Something like:
```suggestion
﻿namespace YAPO.Upgrade.Multipath.Enum
```
Could be extended by using troop, but I guess Upgrade may be verbose enough

Reply:
Is fair